hydpy/models/whmod/whmod_script.py

```diff
@@ -1634,31 +1685,43 @@ def _get_coordinatedict(nodes: hydpy.Nodes) -> Dict[str, _XY]:
         return {n.name: node2xy[n] for n in nodes}
 
     # Conv-Modell PET
-    conv_pet = hydpy.prepare_model("conv_v002")
+    if interpolation_method == "IDW":
+        conv_pet = hydpy.prepare_model("conv_v002")
+    elif interpolation_method == "NN":
+        conv_pet = hydpy.prepare_model("conv_v001")
+    else:
+        hydpy.core.objecttools.assert_never(interpolation_method)
```

Comment:
`assert_never` is now included in Python's `typing` module (and in `typing_extensions`).  Hence, I removed this function from HydPy.